### PR TITLE
Add mpTelemetry-2.0 logs Configurations

### PR DESCRIPTION
**Pull request by pgunapal**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Relates to #27711 
- Changed the OpenTelemetryLogHandler to be a SynchronousHandler
- Added trace and ffdc log source support
- Added server configurations to mpTelemetry-2.0 logging
- Use the ComponentMetadata to retrieve the OpenTelemetry instance, instead of appName
- Added new FAT to test message, trace, and ffdc log sources

### CODE REVIEW

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+mpTelemetryLogging=MicroProfile Telemetry 2.0
+mpTelemetryLogging.desc=MicroProfile Telemetry Logging gathers data from various sources and forwards the data to the configured MicroProfile Telemetry Log Exporter.
+
+source=Source
+source.desc=Specifies a list of comma-separated sources that are used by MicroProfile Telemetry. Valid values are message, trace, ffdc.
```
Comment (is273) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/resources/OSGI-INF/l10n/metatype.properties]: Specifies a list of comma-separated sources for the MicroProfile Telemetry. Valid values for the source attribute are message, trace, and ffdc.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 2]
```diff
@@ -49,4 +49,8 @@ CWMOT5003.factory.used.after.shutdown.useraction=Review the application to see w
...
-CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why it attempted to use MicroProfile Telemetry when there was no application metadata and server scoped OpenTelemetry object. The most likely case is that you are creating new threads without using Liberty's ManagedExecutorService and trying to acquire an OpenTelemetry object from within. If you are attempting to acquire an OpenTemetry object outside an application, ensure OpenTelemetry is enabled via an environment variable or system property.   
\ No newline at end of file
+CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why it attempted to use MicroProfile Telemetry when there was no application metadata and server scoped OpenTelemetry object. The most likely case is that you are creating new threads without using Liberty's ManagedExecutorService and trying to acquire an OpenTelemetry object from within. If you are attempting to acquire an OpenTemetry object outside an application, ensure OpenTelemetry is enabled via an environment variable or system property.
```
Comment (is273) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Review your code to see why the integration code attempted to use MicroProfile Telemetry without metadata and a server scoped OpenTelemetry object. The integration code most likely tries to create new threads without using the ManagedExecutorService interface and to acquire an OpenTelemetry object from within the application. To acquire an OpenTemetry object outside of an application, enable OpenTelemetry through an environment variable or a system property.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 3]
```diff
@@ -49,4 +49,8 @@ CWMOT5003.factory.used.after.shutdown.useraction=Review the application to see w
...
\ No newline at end of file
+CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why it attempted to use MicroProfile Telemetry when there was no application metadata and server scoped OpenTelemetry object. The most likely case is that you are creating new threads without using Liberty's ManagedExecutorService and trying to acquire an OpenTelemetry object from within. If you are attempting to acquire an OpenTemetry object outside an application, ensure OpenTelemetry is enabled via an environment variable or system property.   
+
+CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: MicroProfile Telemetry Logging feature does not recognize the specified {0} log source.
```
Comment (is273) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: The MicroProfile Telemetry logging feature does not recognize the {0} log source.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 4]
```diff
@@ -49,4 +49,8 @@ CWMOT5003.factory.used.after.shutdown.useraction=Review the application to see w
...
+CWMOT5005.mptelemetry.unknown.log.source.explanation=The specified log source is not one of the log sources provided by the Liberty runtime environment.
```
Comment (is273) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: The Liberty runtime environment does not provide the specified log source.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 5]
```diff
@@ -49,4 +49,8 @@ CWMOT5003.factory.used.after.shutdown.useraction=Review the application to see w
...
+CWMOT5005.mptelemetry.unknown.log.source.useraction=Check the configuration information. Ensure that you have specified one of the supported sources for MicroProfile Telemetry.
```
Comment (is273) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Check the configuration information and specify one of the supported sources for MicroProfile Telemetry.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/resources/OSGI-INF/l10n/metatype.properties]: Fixed.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops (hunk 2 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Fixed.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops (hunk 3 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Fixed.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops (hunk 4 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Fixed.

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops (hunk 5 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Fixed.

**Review by is273 — DISMISSED** [review]
LGTM

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/HTTPMetricAdapter.java [hunk 6]
```diff
@@ -26,5 +26,5 @@ public interface HTTPMetricAdapter {
 	 * @param duration
 	 * @param appName This value can be `null` to indicate that we're updating metrics relating to the server. Not all Implementations may need the appname value.
```
Comment (fmhwong) [inline dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/HTTPMetricAdapter.java]: Remove `appName` param in the javadoc.

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/MetricsManager.java [hunk 7]
```diff
@@ -69,7 +69,7 @@ public static MetricsManager getInstance() {
      * @param appName The appName, can be null to indicate server context
      */
 	public void updateHttpMetrics(HttpStatAttributes httpStatAttributes, Duration duration, String appName) {
```
Comment (fmhwong) [inline dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/MetricsManager.java]: The `appName` in the parameter is not used.

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/HTTPMetricAdapter.java (hunk 6 above)
Comment (pgunapal) [reply dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/HTTPMetricAdapter.java]: Fixed.

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/MetricsManager.java (hunk 7 above)
Comment (pgunapal) [reply dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/MetricsManager.java]: Fixed.

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

**Comment by tonyreigns** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+            extName = extKey.substring(extStartIdx + 1);
+        }
+
+        // Map extension name using OTel Attributes naming convention
+        sb.append(MpTelemetryLogFieldConstants.IO_OPENLIBERTY_EXT_TAG).append(extName.toLowerCase());
+
+        return sb.toString();
+    }
+
+    private static String removeSpace(String s) {
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java:262]: Does this method remove spaces after new line character?  Add a comment may help.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java [hunk 9]
```diff
@@ -13,266 +13,287 @@
...
+    public static final String COMPONENT_NAME = "io.openliberty.microprofile.telemetry20.logging.internal.OpenTelemetryLogHandler";
+
+    protected static volatile boolean isInit = false;
+
+    protected CollectorManager collectorMgr = null;
+
+    // Config Source attribute name
+    private static final String SOURCE_LIST_KEY = "source";
+
+    private OpenTelemetryInfo openTelemetry;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java:57]: The variable name is confusing.  It is an instance of `OpenTelemetryInfo` rather than `OpenTelemetry`.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java [hunk 10]
```diff
@@ -13,266 +13,287 @@
...
+            collectorMgr.unsubscribe(this, convertToSourceIDList(sourcesToRemove));
+
+            //Sources to Add -> In New Sources, the difference between newSource and oldSource
+            ArrayList<String> sourcesToAdd = new ArrayList<String>(newSources);
+            sourcesToAdd.removeAll(oldSources);
+            collectorMgr.subscribe(this, convertToSourceIDList(sourcesToAdd));
+
+            sourcesList = newSources; //new primary sourcesList
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java:116]: Change to Tr.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java [hunk 11]
```diff
@@ -13,266 +13,287 @@
...
+
+    /** {@inheritDoc} */
+    @Override
+    public void init(CollectorManager collectorManager) {
+        try {
+            this.collectorMgr = collectorManager;
+            collectorMgr.subscribe(this, convertToSourceIDList(sourcesList));
+            isInit = true;
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java:129]: Change to Tr.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java [hunk 12]
```diff
@@ -13,266 +13,287 @@
...
+        String eventType = MpTelemetryLogMappingUtils.getLibertyEventType(source);
+
+        // Map the Liberty event to the Open Telemetry Logs Data Model
+        MpTelemetryLogMappingUtils.mapLibertyEventToOpenTelemetry(builder, eventType, event);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void synchronousWrite(Object event) {
+        OpenTelemetryInfo otelInstance = null;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java:153]: Variable name is confusing.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java [hunk 13]
```diff
@@ -13,266 +13,287 @@
...
+
+    /*
+     * Given the sourceList in 'config' form, return the sourceID
+     * Proper sourceID format is <source> | <location>
+     */
+    private List<String> convertToSourceIDList(List<String> sourceList) {
+        List<String> sourceIDList = new ArrayList<String>();
+        for (String source : sourceList) {
+            String sourceName = getSourceName(source);
+            if (!sourceName.equals("")) {
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java:249]: Consider to use `isEmpty()`

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/package-info.java [hunk 14]
```diff
@@ -8,7 +8,7 @@
  * SPDX-License-Identifier: EPL-2.0
  *
  * Contributors:
- *     IBM Corporation - initial API and implementation
+ * IBM Corporation - initial API and implementation
  *******************************************************************************/
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/package-info.java:12]: Is this change needed?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/bnd.bnd [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+bVersion=1.0
+
+javac.source: 11
+javac.target: 11
+
+fat.minimum.java.level: 11
+
+src: \
+        fat/src,\
+        test-applications/ffdc-servlet/src, \
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/bnd.bnd]: Remove trailing `, \`

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/build.gradle [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/build.gradle:12]: Is this intentional to be an empty file?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java [hunk 17]
```diff
@@ -0,0 +1,114 @@
...
+    @ExpectedFFDC("java.lang.ArithmeticException")
+    public void testTelemetryMessages() throws Exception {
+        hitWebPage("ffdc-servlet", "FFDCServlet", true, "?generateFFDC=true");
+
+        String logLevelLine = server.waitForStringInLog(".*scopeInfo.*", server.getConsoleLogFile());
+        String exceptionTraceLine = server.waitForStringInLog("exception.message=\"divide by zero\"", server.getConsoleLogFile());
+        String exceptionMessageLine = server.waitForStringInLog("exception.stacktrace=\"java.lang.ArithmeticException: divide by zero", server.getConsoleLogFile());
+        String exceptionTypeLine = server.waitForStringInLog("exception.type=\"java.lang.ArithmeticException\"", server.getConsoleLogFile());
+
+        assertTrue("FFDC Log levelwas not logged by MPTelemetry", logLevelLine.contains("WARN "));
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java]: `levelwas` to `level was`

File: dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java [hunk 18]
```diff
@@ -91,11 +91,15 @@ public OpenTelemetryInfoFactoryImpl(@Reference MetaDataSlotService slotService,
         this.openTelemetryVersionedConfiguration = openTelemetryVersionedConfiguration;
 
         OpenTelemetry runtimeInstance = this.openTelemetryVersionedConfiguration.createServerOpenTelemetryInfo(getServerTelemetryProperties());
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java:93]: I'd name the variable `openTelemetry` rather than `runtimeInstance`.  Very confusing to read.

**Review by fmhwong — CHANGES_REQUESTED** [review]
Need to handle the exceptions instead of just `e.printStackTrace()`.  A few minor variable naming suggestions.  We are just testing logging exporter.  Are you planning to add tests to export using otlp and check the result from a OTel collector?

**Comment by tonyreigns** [comment]
> Need to handle the exceptions instead of just `e.printStackTrace()`. A few minor variable naming suggestions. We are just testing logging exporter. Are you planning to add tests to export using otlp and check the result from a OTel collector?

Right now we just wanted a few basic tests but we'll have quite a few more tests after this beta.

**Comment by tonyreigns** [comment]
All the above issues will be resolved post beta so we can guarantee the changes to be merged.

Nothing needs to be handled in the try/catches, this is how it's being done for JSONHandler. The exceptions will be properly printed using TR. The rest will also be cleaned up. Thanks.

**Review by fmhwong — DISMISSED** [review]
Review comments to be addressed in a follow up PR.

**Review by tonyreigns — DISMISSED** [review]

**Comment by tonyreigns** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

**Comment by tonyreigns** [comment]
Test failed due to the test machines having different messages than local machine...

On mac, expected error is:` exception.message="divide by zero",`
On the test machine, this error was returned:  `exception.message="/ by zero"`

**Comment by tonyreigns** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

**Comment by tonyreigns** [comment]
#build

**Review by tonyreigns — APPROVED** [review]

**Comment by tonyreigns** [comment]
The failing test was disabled so we can merge this PR and will be reenabled in another PR. The builds are all green otherwise aside from another known defect with MPTelemetry 1.1(RTC 300893)

The FFDC test failed due to the test machines having different messages than local machine...

On mac, expected error is:` exception.message="divide by zero",`
On the test machine, this error was returned:  `exception.message="/ by zero"`
